s3parq/publish_parq.py

```diff
@@ -217,28 +237,10 @@ def publish(bucket: str, key: str, partitions: iter, dataframe: pd.DataFrame, re
         published_files = _assign_partition_meta(bucket=bucket,
                                                  key=key,
                                                  dataframe=dataframe,
-                                                 partitions=partitions)
+                                                 partitions=partitions,
+                                                 session_helper=session_helper, redshift_params=redshift_params)
```

Comment:
Can this be set to next line just to match up?

Reply:
Done!